data/sql/updates/pending_db_world/rev_1638401383244389098.sql

```diff
@@ -0,0 +1,14 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1638401383244389098');
+
+DROP TABLE IF EXISTS `playercreateinfo_cast_spell`;
+CREATE TABLE IF NOT EXISTS `playercreateinfo_cast_spell` (
+  `raceMask` INT UNSIGNED NOT NULL DEFAULT '0',
+  `classMask` INT UNSIGNED NOT NULL DEFAULT '0',
+  `spell` MEDIUMINT UNSIGNED NOT NULL DEFAULT '0',
+  `note` VARCHAR(255) DEFAULT NULL
```

Comment:
@Kitzunu is this one okay?

Reply:
Kitzunu got with me and provided me the link to the guide.
https://www.azerothcore.org/wiki/sql-standards#tables-and-columns
i ensure it was with in compliance.